tdp/cli/commands/deploy.py

```diff
@@ -62,6 +64,12 @@
 )
 @click.option("--filter", type=str, help="Glob on list name")
 @click.option("--dry", is_flag=True, help="Execute dag without running any operation")
+@click.option(
```

Comment:
Does this mean we need to pass the deployment id anytime we want to resume ? Can't we resume automatically if previous deployment was a failure ?

Reply:
We can do this, would a `get_last_deployment` with a query like `select(DeploymentLog).order_by(DeploymentLog.id.desc()).first()` be fine?

Reply:
Yes